openquake/server/views.py

```diff
@@ -376,7 +394,7 @@ def get_traceback(request, calc_id):
 
 @cross_domain_ajax
 @require_http_methods(['GET'])
-def get_result(request, result_id):
+def get_result_ng(request, calc_id, result_id):
     """
     Download a specific result, by ``result_id``.
 
```

Comment:
What's the point of get_result_ng? If the idea was to restrict the accepted result_ids to the ones associated to calc_id, then the check must be implemented. If we want to leave the check for the next release, then let's implement get_result_ng in the next release. If the idea is to forbid the user from seeing the results of another user, then this is an issue of permissions which require some discussion and which I would leave to the next sprint.